pkg/handlers/aws/mutation/iaminstanceprofile/variables.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2023 D2iQ, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package iaminstanceprofile
```

Comment:
this variable's discovery does not needed as it is discovered part of aws node spec. The unit test for this variable can be added to `pkg/handlers/aws/clusterconfig/variables_test.go`

Reply:
Ahh now I see what you mean, thanks! Yeah lets remove this unnecessary code https://github.com/d2iq-labs/capi-runtime-extensions/pull/216/commits/c6b71c7cb7941c50e7d91aa0cde6d415d62066f1